### PR TITLE
bump version to 1.4.0

### PR DESCRIPTION
**Pull request by bvered**
Signed-off-by: 662962756 <bvered@il.ibm.com>

### CODE REVIEW

File: docs/docs_general/docs_whatsnew.dita [hunk 1]
```diff
@@ -20,7 +20,7 @@ translate="no">ODF FlashSystem driver</ph>.</shortdesc>
 <entry colname="col1">31 January 2023</entry>
 <entry colname="col2">
 <p>The version information was added in IBM Documentation as part of the initial <ph translate="no"
->ODF FlashSystem driver</ph> 1.3.0 release.</p>
+>ODF FlashSystem driver</ph> 1.4.0 release.</p>
```
Comment (tyichye19) [inline docs/docs_general/docs_whatsnew.dita:23]: I think it is better to update the doc files altogether, and not just the version. Now we just have incorrect docs (1.4.0 release date, 1.4.0 does not add support for multiple storage systems, etc.)

File: go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/IBM/ibm-storage-odf-block-driver
 go 1.18
 
 require (
-	github.com/IBM/ibm-storage-odf-operator v1.3.0
+	github.com/IBM/ibm-storage-odf-operator v1.3.1-0.20230201081526-a62cc98b3818
```
Comment (tyichye19) [inline go.mod:6]: is that the tag we get for operator 1.4.0?

Comment (bvered) [reply go.mod:6]: yep, I'll try to update the tag after changing the operator.

File: docs/docs_general/docs_whatsnew.dita (hunk 1 above)
Comment (bvered) [reply docs/docs_general/docs_whatsnew.dita]: reverting documentation files

**Review by tyichye19 — APPROVED** [review]
LGTM

**Pull request merged**